napari/layers/base/base.py

```diff
@@ -597,6 +598,13 @@ def _dims_displayed(self):
         # for additional discussion.
         return self._dims_order[-self._ndisplay :]
 
+    @property
+    def _dims_displayed_mask(self):
+        """To be removed displayed dimensions mask"""
```

Comment:
I find this name and docstring hard to parse. Can you clarify a bit what this means?

Reply:
agree it's not the easiest to parse but not immediately sure of a better name... will improve the docstring

I'm not sure if this will still be removed as layer data sources are separated from layers but will leave until that happens 👍🏼 
```suggestion
    @property
    def _dims_displayed_mask(self):
        """**to be removed** displayed dimensions mask"""
```